### DISCUSSION TITLE
ebook support

### DISCUSSION THREAD

**Discussion by Emiliaaah** [LeoKlaus/plappa#16] [Feature Requests]
I really like the aesthetic of the app, the only feature I'm missing for audiobookshelf as the server (in my case) is ebook support. It would be great if I can listen and read ebooks through the same aesthetically pleasing app. This is the only big thing stopping me from switching to this app right this second.

**Comment by LeoKlaus** [comment]
Pretty cool idea, it's probably a bit of work but I'd really like to do this!

**Reply by niwena** [reply]
would love to see this realized.

**Reply by itHINaSu** [reply]
I also would love to cast a vote to see this.

**Reply by jeeftor** [reply]
Dumb solution -> use PLAPPA to download the ebook and drop into IOSs "books" app which already has ebook support :)

**Comment by jeeftor** [comment]
I just use my Kindle paperwhite for books and this for audiobooks. I don't really see an overall benefit to this unless my ebook and audiobook progress could be synced .
